db/migrate/20190214125845_add_comments_to_miq_tasks_table.rb

```diff
@@ -0,0 +1,21 @@
+class AddCommentsToMiqTasksTable < ActiveRecord::Migration[5.0]
+  def up
+    change_table_comment :miq_tasks, "ManageIQ background tasks"
+
+    change_column_comment :miq_tasks, :context_data, "Serialized metadata that may contain more detailed information about the task."
+    change_column_comment :miq_tasks, :created_on, "[builtin] The timestamp the record was added to the app inventory."
+    change_column_comment :miq_tasks, :id, "[builtin] The internal database ID."
```

Comment:
it should be "... record id", may be specify that this column used as the primary key

Reply:
Ok, done.